src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs

```diff
@@ -715,5 +717,14 @@ private void SetCheckboxSelectionState(object item, ListViewItem? lviContainer =
 					checkbox.IsChecked = FileList.SelectedItems.Contains(item);
 			}
 		}
+
+		private void TagItem_Tapped(object sender, TappedRoutedEventArgs e)
+		{
+			var tagName = ((sender as StackPanel)?.Children[TAG_TEXT_BLOCK] as TextBlock)?.Text;
+			if (tagName is null)
```

Comment:
When can it be null?

Reply:
I might remove it. I added it to avoid future problems with refactoring (like if we add a third control that is not a textblock)

Reply:
Thanks, good future proofing.